src/components/Pages/Student/StudentGroup.vue

```diff
@@ -81,10 +92,29 @@ export default {
         ...mapActions('groups', {
             findGroups: 'find',
         }),
+
+        ...mapActions('users', {
+            findStudent: 'find',
+        }),
+
+        findStudentsInGroup(group) {
```

Comment:
Since this returns students' _count_, not students themselves, I'd prefer the method name to correctly reflect that.